vyper/cli/vyper_compile.py

```diff
@@ -106,6 +111,7 @@ def _parse_args(argv):
     )
     parser.add_argument("--no-optimize", help="Do not optimize", action="store_true")
     parser.add_argument("--optimize", help="Optimization flag", choices=["gas", "codesize", "none"])
```

Comment:
Actually, I think you should add here the info to the choices what is the default, i.e. `gas`.

Reply:
fixed in 8b98bcd0b